shared/common-adapters/user-proofs.desktop.js

```diff
@@ -9,7 +9,7 @@ import {metaNone, checking as proofChecking} from '../constants/tracker'
 
 import type {Proof, Props, MissingProof} from './user-proofs'
 
-export function MissingProofRow (proof: MissingProof, style: Object): React$Element {
+function MissingProofRow (proof: MissingProof, style: Object): React$Element<*> {
```

Comment:
I would love to have a reason for it. It's tied to the transition to flow 0.29. Marco just told me it's called "Existential Types" AKA "I don't know what this is but you should be able to figure it out flow" -Marco